### ISSUE TITLE
Required Python version is very restrictive

### ISSUE THREAD

**Issue by PabRod** [parallelwindfarms/byteparsing#21]
## Problem

The expected Python version for installing this package is `python = ">=3.9,<3.11"`. We don't need to be that restrictive anymore.

## Solution

Expand the versions range.

**Comment by PabRod** [comment]
Related to #16

**Comment by jhidding** [comment]
Yes, the problem is not that it won't work on other versions. It's just not been tested. I removed the `setup.py` in favour of using only the `pyproject.toml`. We should be fine with anything above Python 3.9. I'm using quite a bit of type annotation, using `list[T]` and the likes. I don't think we should go below 3.9.
